Jenkinsfile

```diff
@@ -105,8 +105,13 @@ pipeline {
                 sh '''
                 . venv/bin/activate
 
-                coverage run --branch --source=./${PROJECT_NAME} --omit=src/utils/hdfs_mods.py,src/utils/wrappers.py,src/utils/runlog.py,src/_version.py,src/pipeline.py \
-              -m pytest -ra ./tests --ignore=tests/test_utils/test_hdfs_mods.py
+                python3 -m pytest --junitxml "junit-report.xml" "./tests"
+                '''
+
+                junit restResults: 'junit-report.xml'
+
+                /* coverage run --branch --source=./${PROJECT_NAME} --omit=src/utils/hdfs_mods.py,src/utils/wrappers.py,src/utils/runlog.py,src/_version.py,src/pipeline.py \
+                    -m pytest -ra ./tests --ignore=tests/test_utils/test_hdfs_mods.py
```

Comment:
I think James or Charlie might have put in this code to omit hdfs mods code in Jenkins. However, Charlie also added newer code so the unit tests for hdfs would be ignored in Jenkins, so hopefully this can go.